ui/mirage/handlers/activity.js

```diff
@@ -20,10 +20,15 @@ export default function (server) {
     };
   });
 
-  server.get('sys/internal/counters/config', function (db) {
+  server.get('sys/internal/counters/config', function () {
     return {
       request_id: '00001',
-      data: db['clients/configs'].first(),
+      data: {
+        default_report_months: 12,
```

Comment:
you can remove the default_report_months as we don't display this anymore (I removed it in a recent PR)

Reply:
It does still come back from the API, so I'm inclined to leave it for as close parity to the real API as we can get. But good to know we don't display, another thing to add to the tests! :D